src/extension/background-script/actions/accounts/edit.ts

```diff
@@ -8,6 +9,21 @@ const edit = async (message: MessageAccountEdit) => {
   if (accountId in accounts) {
     accounts[accountId].name = message.args.name;
 
+    if (
```

Comment:
This feels strange, it only applies to alby and the call to edit has to pass a fake message. I would not call this method, instead, edit these fields from within the alby connector.